docs/advanced-features/custom-error-page.md

```diff
@@ -98,3 +98,4 @@ If you have a custom `Error` component be sure to import that one instead. `next
 ### Caveats
 
 - `Error` does not currently support Next.js [Data Fetching methods](/docs/basic-features/data-fetching.md) like [`getStaticProps`](/docs/basic-features/data-fetching/get-static-props.md) or [`getServerSideProps`](/docs/basic-features/data-fetching/get-server-side-props.md).
+- `/_error` is a reserved pathname and will render 404 when accessed directly via [routing](/docs/routing/introduction) or rendering in a [custom server](/docs/advanced-features/custom-server). You should create another page if you need.
```

Comment:
```suggestion
- `_error.js`, like `_app.js`, is a reserved page that is used to define the customized layouts and behaviors of the error pages. `/_error` will render 404 when accessed directly via [routing](/docs/routing/introduction) or rendering in a [custom server](/docs/advanced-features/custom-server).
```

IMHO the documentation should not only mention "it is not a catch-all error page" but also mention what it actually does. Comparing `_error` with `_app` gives developers a better idea about what it is (and then know why it shouldn't be used as a page).

Reply:
Agree